### PR TITLE
more ECFC L2CAP fixes

### PR DESCRIPTION
**Pull request by sjanc**

### CODE REVIEW

File: ptsprojects/zephyr/l2cap.py [hunk 1]
```diff
@@ -135,6 +135,8 @@ def test_cases(ptses):
                   "L2CAP", "TSPX_psm_unsupported", format(psm_unsupported, '04x'))),
               TestFunc(lambda: pts.update_pixit_param(
                   "L2CAP", "TSPX_iut_supported_max_channels", "2")),
+              TestFunc(lambda: pts.update_pixit_param(
+                  "L2CAP", "TSPX_l2ca_num_concurrent_credit_based_connections", "2")),
```
Comment (michal-narajowski) [inline ptsprojects/zephyr/l2cap.py:139]: Is this typo intentional?

File: wid/l2cap.py [hunk 2]
```diff
@@ -599,6 +599,28 @@ def hdl_wid_262(desc):
...
+    stack = get_stack()
+    l2cap = stack.l2cap
+
+    btp.l2cap_conn(None, None, l2cap.psm, l2cap.initial_mtu)
+    return True
+
+
+def hdl_wid_268(desc):
+    # PTS ask to disconnect channel with specified SCID and DCID but we can just
+    # disconnect last connected channel (1) to be confirming with TS
```
Comment (michal-narajowski) [inline wid/l2cap.py:614]:
```suggestion
    # disconnect last connected channel (1) to be conforming with TS
```

File: wid/l2cap.py [hunk 3]
```diff
@@ -599,6 +599,28 @@ def hdl_wid_262(desc):
...
+
+    btp.l2cap_disconn(1)
+
+    # Wait for disconnected event before replying to PTS to avoid sending next
+    # connect to soon
```
Comment (michal-narajowski) [inline wid/l2cap.py:619]:
```suggestion
    # connect too soon
```

**Review by michal-narajowski — CHANGES_REQUESTED** [review]

File: ptsprojects/zephyr/l2cap.py (hunk 1 above)
Comment (sjanc) [reply ptsprojects/zephyr/l2cap.py:139]: yeap :)

**Review by michal-narajowski — APPROVED** [review]

**Pull request merged**